src/components/molecules/CarouselText/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './style.css';
+import Title from '../../atoms/Title/index';
+
+function CarouselText() {
```

Comment:
is it reusable component?

Reply:
changed it, now we can use it with different text